bundles/EcommerceFrameworkBundle/Resources/public/js/bundle.js

```diff
@@ -132,7 +132,7 @@ pimcore.bundle.EcommerceFramework.bundle = Class.create(pimcore.plugin.admin, {
         var that = this;
 
         Ext.Ajax.request({
-            url: '/admin/ecommerceframework/reports/piwik/reports',
+            url: Routing.generate('pimcore_ecommerceframework_reports_piwik_reports'),
```

Comment:
Bundle loading failed with route `pimcore_ecommerceframework_reports_piwik_reports` does not exist error
![image](https://user-images.githubusercontent.com/5137917/89272855-0ac22200-d63f-11ea-9e85-5f5c2a3ea391.png)

Reply:
fixed